src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,62 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    public static final int ACTION_PART_INDEX = 0;
+    public static final int NUMBERS_PART_INDEX = 1;
+    public static final int SUPPLY_PART_INDEX = 0;
+    public static final int BUY_PART_INDEX = 1;
+    public static final String SUPPLY_STRING = "supply";
+    public static final String SEPARATION_PLACE = ",";
+    public static final String BUY_STRING = "buy";
+    public static final String REPORT_STRING = "result";
+
+    private String[] readFromFile(String fileName) {
+        ArrayList<String> linesList = new ArrayList<>();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName))) {
+            String line;
+            while ((line = bufferedReader.readLine()) != null) {
+                linesList.add(line);
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file", e);
+        }
+        return linesList.toArray(new String[0]);
```

Comment:
It's better to use a List<String> as the return type for the 'readFromFile' method. Returning an array forces an unnecessary conversion from List to array.